app/src/main/java/com/mmm/parq/fragments/DriverHomeFragment.java

```diff
@@ -79,10 +81,10 @@
     private static String CLASS = "DriverHomeFragment";
 
     public enum State {
-        FIND_SPOT, NAVIGATION, OCCUPY_SPOT, END_RESERVATION
+        FIND_SPOT, NAVIGATION, ARRIVE_SPOT, OCCUPY_SPOT, END_RESERVATION
```

Comment:
it would be nice if we sat down and created some really solid state names and then made them constant across platforms